32blit.toolchain

```diff
@@ -1,8 +1,8 @@
 set(CMAKE_SYSTEM_NAME Generic)
 set(CMAKE_SYSTEM_PROCESSOR arm)
 
-set(CMAKE_C_COMPILER arm-none-eabi-gcc)
-set(CMAKE_CXX_COMPILER arm-none-eabi-g++)
+set(CMAKE_C_COMPILER arm-none-eabi-gcc CACHE PATH "Path to C compiler")
```

Comment:
You've got me at a loss here (my knowledge of CMake is still pretty limited). Is `"Path to C compiler"` a literal value understood by `CACHE PATH` or is this a value that the user needs to supply (by manually editing this file) before running `cmake ..`?

Reply:
It's just documentation for the variable. I think it's used by the GUI.